File: src/SampleCRM.Web/Services/CountService.cs
using OpenRiaServices.DomainServices.Hosting;
using OpenRiaServices.DomainServices.Server;
using SampleCRM.Web.Models;
using System.Linq;

namespace SampleCRM.Web
{
    [EnableClientAccess]
    public class CountService : SampleCRMService
    {
        [Query]
        public IQueryable<CountModel> GetAllCounts() => 
            new CountModel[] { new CountModel
                {
                    OrderCount = _context.Orders.Count(),
                    CustomerCount = _context.Customers.Count(),
                    CategoryCount = _context.Categories.Count(),
                    ProductCount = _context.Products.Count()
                }
            }.AsQueryable();
    }
}

File: src/SampleCRM.Web/Services/CountService.vb
Imports OpenRiaServices.DomainServices.Hosting
Imports OpenRiaServices.DomainServices.Server
Imports SampleCRM.Web.Models
Imports System.Linq

Namespace SampleCRM.Web
    <EnableClientAccess>
    Public Class CountService
        Inherits SampleCRMService

        <Query>
        Public Function GetAllCounts() As IQueryable(Of CountModel)
            Dim countModel = New CountModel With {
            .OrderCount = _context.Orders.Count(),
            .CustomerCount = _context.Customers.Count(),
            .CategoryCount = _context.Categories.Count(),
            .ProductCount = _context.Products.Count()
        }
            Return New CountModel() {countModel}.AsQueryable()
        End Function
    End Class
End Namespace
